src/cli/main/cli.py

```diff
@@ -411,11 +411,16 @@ class Config(NamedTuple):
     cache: List[Dict[str, str]]
 
 
+def _get_named_temporary_file_name() -> str:
```

Comment:
@rohaquinlop Sometimes we get an error saying that a temporary directory already exists when we run several instances of Makes at the same time. Can you please extend this function so it waits 1 second and retries if the directory already exists? Let's retry 5 times before failing.

Reply:
Related issue is https://github.com/fluidattacks/makes/issues/1068

Reply:
Ok, got it